docker/Dockerfile.frontend

```diff
@@ -3,6 +3,4 @@ WORKDIR /app
 ENV PATH /app/node_modules/.bin:$PATH
 COPY frontend/package.json /app/package.json
 COPY frontend/package-lock.json /app/package-lock.json
-RUN npm install --silent
-RUN npm install react-scripts@3.0.1 -g --silent
-RUN npm install node-sass@latest
+RUN npm install
```

Comment:
You should usually run this with `--production` flag so no dev dependencies are installed. Also keep in mind that `npm install` can overwrite `package-lock` if there are newer packages versions than specified in `package.json`. It is better to use a dependency manager that respects lockfiles like `yarn`. This problem probably won't affect you but it's good to know.

Reply:
This build is development-oriented, though I will take this into account while making the production one. Thanks!